pages/public_cloud/compute/create_a_public_cloud_project/guide.fr-fr.md

```diff
@@ -18,31 +18,31 @@ La création d’un projet est la première étape indispensable pour déployer
 
 Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr), et cliquez sur le menu `Public Cloud`{.action}.
 
-Cliquez ensuite sur `Créez votre premier projet OVH Public Cloud`{.action}.
+Après en avoir pris connaissance, validez les termes des contrats en cochant la case correspondante puis cliquez sur `Découvrir l'univers Public Cloud`{.action}.
 
-Si vous disposez d'un bon d'achat, vous pouvez alors le renseigner. Seuls les bons d'achats fournis pour votre filiale OVHcloud peuvent être utilisés.
+![création de projet](images/firstproject2024.png){.thumbnail}
 
-> [!warning]
-> Si vous aviez précédemment créé un projet Public Cloud sur votre compte et que vous l'avez supprimé, vous ne pourrez utiliser le bon d'achat.
->
-
-Donnez un nom à votre projet, prenez connaissances des contrats et acceptez-les, puis cliquez sur le bouton `Continuer`{.action}.
-
-![name project](images/confirmvoucher.png){.thumbnail}
-
-Vous serez alors invité à **enregistrer un mode de paiement** si vous n’en avez pas déjà renseigné un. Celui-ci vous sera demandé même si un bon d'achat est appliqué.  
+- Si vous n'avez pas encore de moyen de paiement valide dans votre compte OVHcloud, le processus de création du projet Public Cloud commence. Votre projet sera alors créé en [Mode découverte](#discovery).
```

Comment:
[Mode découverte](#discovery) n'est pas un lin ancre valide, c'est valable dans toute les langues il me semble.

Reply:
It does work on the help centre. It redirects to explanations at the bottom of the page.